.github/workflows/finishReleaseCycle.yml

```diff
@@ -28,16 +36,19 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.OS_BOTIFY_TOKEN }}
           ISSUE_NUMBER: ${{ github.event.issue.number }}
           COMMENT: |
-            This issue either has unchecked QA steps or has not yet been marked with the `:shipit:` emoji of approval.
+            Oops, something went wrong! This issue must:
+                1. Have unchecked QA steps, or
+                2. Not have been marked with the `:shipit:` emoji of approval, or
+                3. Have been closed by someone who is not a member of @Expensify/Mobile-Deployers.
```

Comment:
Can the issue still be closed by QA to trigger the deploy?

Reply:
Nope